site/src/components/WorkspaceDeletedBanner/WorkspaceDeletedBanner.tsx

```diff
@@ -22,22 +22,20 @@ export const WorkspaceDeletedBanner: FC<React.PropsWithChildren<WorkspaceDeleted
 }) => {
   const styles = useStyles()
 
-  if (!isWorkspaceDeleted(workspace)) {
-    return null
-  }
-
   return (
-    <Alert
-      className={styles.root}
-      action={
-        <Button color="inherit" onClick={handleClick} size="small">
-          {Language.createWorkspaceCta}
-        </Button>
-      }
-      severity="warning"
-    >
-      <AlertTitle>{Language.bannerTitle}</AlertTitle>
-    </Alert>
+    <Maybe condition={workspace.latest_build.status === "deleted"}>
+      <Alert
```

Comment:
No change to the `Alert` here, just wrapping in a `Maybe` and getting rid of the `isWorkspaceDeleted` call